workitem/field_definition_blackbox_test.go

```diff
@@ -36,3 +25,53 @@ func TestListFieldDefMarshalling(t *testing.T) {
 		t.Errorf("field should be %v, but is %v", def, unmarshalled)
 	}
 }
+
+func TestFieldDefinition_Marshalling(t *testing.T) {
```

Comment:
well, I think it's worth adding such a test in order to verify all cases. At least this will provide us with an harness in case of refactoring.